ts/editor/SecondaryInputBadge.svelte

```diff
@@ -9,15 +9,16 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     import * as tr from "../lib/ftl";
     import { getPlatformString, registerShortcut } from "../lib/shortcuts";
     import { context as editorFieldContext } from "./EditorField.svelte";
-    import { htmlOff, htmlOn } from "./icons";
+    import { plainTextIcon, richTextIcon } from "./icons";
 
     const editorField = editorFieldContext.get();
     const keyCombination = "Control+Shift+X";
     const dispatch = createEventDispatcher();
 
     export let off = false;
+    export let defaultInput: "plain" | "rich" = "rich";
 
-    $: icon = off ? htmlOff : htmlOn;
+    $: icon = defaultInput == "rich" ? plainTextIcon : richTextIcon;
```

Comment:
I think you could leave this as the `PlainTextBadge`.
It seems a bit silly to generalize this component to serve as both the badge for the plain text and the rich text, while still having other badges like the sticky badge, which also mostly look the same.